proto/osmosis/validator-preference/v1beta1/genesis.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package osmosis.validatorpreference.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "osmosis/validator-preference/v1beta1/params.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v12/x/validator-preference/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// GenesisState defines the validator-set preference module's genesis state.
+message GenesisState {
```

Comment:
Do we not need to add genesis state for the val set preferences state?

Reply:
I was thinking we need genesis state to set ValSet fee during module initialization. Is that not the case?